volatility3/framework/plugins/layerwriter.py

```diff
@@ -114,10 +120,9 @@ def _generator(self):
                             progress_callback=self._progress_callback,
                         )
                         file_handle.close()
+                        output_name = file_handle._output_filename
```

Comment:
Accessing a private member is bad.  If it's supposed to be exposed it should be public.

Reply:
hm.. that's not a private member (_ vs __) but you are absolutely right that in the case we want to access to that filename we meed to make it public

Reply:
Yeah, I meant protected going by java nomenclature, but basically means should only be accessed from within the class...